From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Aditya Garg <gargaditya08@live.com>
Date: Sun, 13 Apr 2025 11:43:53 +0530
Subject: Fix sparse errors

apple_bce.h was using #pragma once, which is not acceptable by sparse
---
 drivers/staging/apple-bce/apple_bce.h | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/drivers/staging/apple-bce/apple_bce.h b/drivers/staging/apple-bce/apple_bce.h
index 111111111111..222222222222 100644
--- a/drivers/staging/apple-bce/apple_bce.h
+++ b/drivers/staging/apple-bce/apple_bce.h
@@ -1,4 +1,5 @@
-#pragma once
+#ifndef APPLE_BCE_H
+#define APPLE_BCE_H
 
 #include <linux/pci.h>
 #include <linux/spinlock.h>
@@ -35,4 +36,6 @@ struct apple_bce_device {
     struct bce_vhci vhci;
 };
 
-extern struct apple_bce_device *global_bce;
\ No newline at end of file
+extern struct apple_bce_device *global_bce;
+
+#endif //APPLE_BCE_H
-- 
Armbian

